RubeesEat/Pages/SplitBill.cshtml

```diff
@@ -1,28 +1,43 @@
 @page
 @inject IPersonRepository PersonRepository
 
-Split bill:
-<form id="splitBillForm" method="post" action="/api/bills">
+<head>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/site.css" />
+</head>
 
-    <label for="billDescription">Description:</label>
-    <input id="billDescription" t-id="billDescription" type="text" name="billDescription" required="required"></input><br>
+<div class="container mt-3">
+    <h1 class="text-center">Split bill:</h1>
+    <form id="splitBillForm" method="post" action="/api/bills" class="mx-auto mt-4" style="max-width: 600px;">
+        <div class="mb-3">
+            <label for="billDescription">Description:</label>
+            <input class="form-control" style="max-width: 900px;" id="billDescription" t-id="billDescription" type="text" name="billDescription" required="required"></input><br>
```

Comment:
I mentioned it before but all the `max-width` styling should be removed in the end. In fact, we don't want any custom styling using the `style` attributes and minimal positioning/margins using the classes.